### DISCUSSION TITLE
vant4 组件演示的demo 原代码在哪里?感谢

### DISCUSSION THREAD

**Discussion by Roooooys** [youzan/vant#13103] [Show and tell]
https://vant.pro 页面右侧的demo源代码开源吗?感谢
![1725425329620](https://github.com/user-attachments/assets/3fb87dba-5eb4-4fc5-a287-dbcbd12683d5)

**Comment by namefhf** [comment]
https://github.com/youzan/vant/blob/main/packages/vant-cli/site/desktop/App.vue
